src/Controller/Frontend/SearchController.php

```diff
@@ -37,12 +37,23 @@ public function __construct(Config $config, Environment $twig, TemplateChooser $
     public function search(ContentRepository $contentRepository, Request $request): Response
     {
         $page = (int) $request->query->get('page', 1);
+        $search = $request->get('search', '');
+        $amountPerPage = $this->config->get('general/listing_records');
 
         // @todo implement actual Search Engine
-        $records = $contentRepository->findForListing($page);
+        if (! empty($search)) {
+            $records = $contentRepository->searchNaive($search, $page, $amountPerPage);
+        } else {
+            $records = null;
+        }
+
+        $context = [
+            'search' => $search,
```

Comment:
rename search to searchQuery or searchedText

Reply:
I'd prefer not. The parameter (in `POST` or `GET`) is also `search`, and it has always been this way historically.

Reply:
`historically` ;) Let's move to the future!
We can break anything from past Bolt versions, can't we?
Also some badly named things?

Reply:
No, we don't have to keep it, but I think it's a properly named thing, in this case. :-)

Reply:
Gentlemen, please :D Variable name "searchValue" exactly says that it is a value which is searching :) "search" means "activity" we do.

Reply:
Alright, alright.. :-)

I've renamed it to `searchTerm` and made the same change in the default templates. I couldn't help myself from keeping the old one backwards compatible, though.